rr_viz/src/rviz_frame.py

```diff
@@ -18,7 +18,7 @@ class RViz_frame():
     # https://stackoverflow.com/questions/7964869/qt-designer-how-to-add-custom-slot-and-code-to-a-button
     # https://doc.qt.io/qt-5/designer-using-custom-widgets.html
     # I did not manager to get this to take a parameter. So, this widget will usually be used in QT via a wrapper widget that sets the paramter.
-    def __init__(self):
+    def __init__(self, parent):
```

Comment:
parent doesn't seem to be used here?

Reply:
its not but if I remove it then the code doesn't work